crates/bevy_asset_store/Cargo.toml

```diff
@@ -0,0 +1,7 @@
+[package]
+name = "bevy_asset_store"
```

Comment:
Should the asset store be limited to just assets? Unity and Unreal can also provide code items as a part of their stores, which for Bevy would likely mean plugins.

Reply:
Yeah, IMO swapping this to `bevy_marketplace_viewer` or something is more clear.